### ISSUE TITLE
EDIF HEX strings

### ISSUE THREAD

**Issue by klammerj** [cliffordwolf/yosys#316]
I was trying to push a pin list through to the edif as a parameter to an iobuf, 
```
#(.LOC("1,2,3,4,5,6,+"))
```
I am now getting those hex strings. 
```
(string "104'h312c322c332c342c352c362c2b")
```
There does not seem a way to differentiate those from other strings. 
(how am I supposed to parse those?)

**Comment by klammerj** [comment]
the is_string property got lost during techmap
here's a patch:
[0001-Fix-for-issue-316.patch.txt](https://github.com/cliffordwolf/yosys/files/790764/0001-Fix-for-issue-316.patch.txt)

**Comment by cliffordwolf** [comment]
Fixed in commit 1e927a5.
